Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -121,6 +122,7 @@ def audit
     return if problem_count.zero?
 
     ofail "#{Formatter.pluralize(problem_count, "problem")} in #{Formatter.pluralize(formula_count, "formula")}"
+    Homebrew.auditing = false
```

Comment:
Can probably skip setting this here as the end of a command is the end of termination, too.